### PR TITLE
Fix translations

### PR DESCRIPTION
**Pull request by jvitkauskas**
Please try and fill out this template where possible, not all fields are required and can be removed.

* **Please check if the PR fulfills these requirements**
- [X] The commit message make sense
- [ ] The changes have been tested locally
- [ ] New features have been documented in the Wiki
- [X] I'm willing to maintain this in the future (Totally Optional)

* **What kind of change does this PR introduce?** 
1. Fixes English translation by correcting syntax errors in json file
1. Updates Lithuanian, Italian translations
1. Adds PowerInput string to other languages to fix Translation.cpp generation

* **What is the current behavior?** 
Translation.cpp could not be generated, because of syntax error in English translation file and missing PowerInput string in other languages.

### CODE REVIEW

File: Translation Editor/translation_it.json [hunk 1]
```diff
@@ -218,6 +218,14 @@
...
 			],
 			"desc": "Calibra le rilevazioni di temperatura attraverso la termocoppia presente nella punta"
+		},
+		"PowerInput": {
+			"text": "PWRW",
+			"text2": [
+				"Power",
+				"Wattage"
+			],
+			"desc": "Power Wattage of the power adapter used"
```
Comment (federck) [inline Translation Editor/translation_it.json]: Imposta la potenza dell'alimentatore in uso

File: Translation Editor/translation_it.json [hunk 2]
```diff
@@ -218,6 +218,14 @@
 				"avanzata"
 			],
 			"desc": "Calibra le rilevazioni di temperatura attraverso la termocoppia presente nella punta"
+		},
+		"PowerInput": {
+			"text": "PWRW",
+			"text2": [
+				"Power",
+				"Wattage"
```
Comment (federck) [inline Translation Editor/translation_it.json]: alimentaz

File: Translation Editor/translation_it.json [hunk 3]
```diff
@@ -218,6 +218,14 @@
 				"avanzata"
 			],
 			"desc": "Calibra le rilevazioni di temperatura attraverso la termocoppia presente nella punta"
+		},
+		"PowerInput": {
+			"text": "PWRW",
+			"text2": [
+				"Power",
```
Comment (federck) [inline Translation Editor/translation_it.json]: Potenza

**Comment by federck** [comment]
I've added translation in comments to make the process faster.
In order: Power→Potenza, Wattage→alimentaz, and "Imposta la potenza..." goes to the description.
Feel free to add them in your repo, thank you anyway.

**Comment by jvitkauskas** [comment]
@federck thanks!

**Comment by Ralim** [comment]
Thanks.
I'm terrible at getting updates working to the translation files :(

**Pull request merged**